src/Post.php

```diff
@@ -673,7 +673,7 @@ public function has_term($term_name_or_id, $taxonomy = 'all')
      */
     public function comment_count()
```

Comment:
I would add return type as well.

Reply:
Resolved in [`97451ef` (#2670)](https://github.com/timber/timber/pull/2670/commits/97451efdb488ca97da192afc46c5006e7938bf3d).